packages/types/api/executions.d.ts

```diff
@@ -1,5 +1,10 @@
 export type ExecutionRecordStatus = 'completed' | 'failed' | 'running' | 'unknown';
 
+export type ExecutionProcessingTimes = {
```

Comment:
ExecutionProcessingTimes type is not used for execution.  Should we move it to granules.d.ts?

Reply:
I just pulled it directly from the rds-phase-2 branch.  I assume they had a reason for wanting it in executions.d.ts?

Reply:
ok